src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/Utility/PackageSourceMappingUtility.cs

```diff
@@ -9,8 +9,8 @@ internal static class PackageSourceMappingUtility
     {
         public static bool IsMappingEnabled(ISettings settings)
         {
-            var packageNamespacesConfiguration = PackageNamespacesConfiguration.GetPackageNamespacesConfiguration(settings);
-            bool isMappingEnabled = packageNamespacesConfiguration?.AreNamespacesEnabled ?? false;
+            var packageSourceMappingConfiguration = PackageSourceMappingConfiguration.GetPackageSourceMappingConfiguration(settings);
+            bool isMappingEnabled = packageSourceMappingConfiguration?.IsEnabled ?? false;
```

Comment:
If we keep previous convention then this one should be `ArePackageSourceMappingsEnabled`?